crates/lumina_client/src/ui/game_ui/boostmeter.rs

```diff
@@ -0,0 +1,50 @@
+use bevy::prelude::*;
+use velyst::prelude::*;
+
+use super::main_ui::GameUi;
+pub(super) struct BoostmeterUiPlugin;
+
+impl Plugin for BoostmeterUiPlugin {
+    fn build(&self, app: &mut App) {
+        app.register_typst_asset::<GameUi>()
+            .compile_typst_func::<GameUi, BoostmeterFunc>()
+            .init_resource::<BoostmeterFunc>()
+            .insert_resource(BoostmeterFunc {
+                height: 10.0,
+                width: 213.0,
+                red_height: 0.0,
+            })
+            .add_systems(Update, update_boost_meter);
+    }
+}
+
+// Update the booster fill state
```

Comment:
```suggestion
/// Update the booster fill state
```
3 slashes create a doc comment instead of a normal comment (will show up in IDE on inspection)